tests/python/frontend/onnx/test_forward.py

```diff
@@ -30,6 +30,30 @@
 import scipy
 
 
+def get_tvm_output_with_vm(graph_def, input_data, target, ctx,
+                           opset=None, mode=None):
+    """ Generic function to execute and get tvm output with vm executor"""
+    if isinstance(input_data, list):
```

Comment:
This if/else is dup from get_tvm_output, move it to a common function

Reply:
Thanks for the suggestions and I updated the part you mentioned.